redshift/helpers.go

```diff
@@ -126,3 +126,15 @@ func isRetryablePQError(code string) bool {
 	_, ok := retryable[code]
 	return ok
 }
+
+func splitCsvAndTrim(raw string) []string {
+	rawSlice := strings.Split(raw, ",")
```

Comment:
It would be much better to use the `encoding/csv` (https://pkg.go.dev/encoding/csv) package for parsing CSV. Commas can also appear inside quoted values and that would break the current code.

Reply:
AFAIK comma is not allowed in an ARN but regardless I'm happy to make this change

Reply:
Okay this change has been pushed up to the PR